### PR TITLE
Add callback param for accessing C3Chart object

### LINKED ISSUE #204: Provide access to the chart object for C3 charts
When using C3 charts is often necessary to access the API for performing additional operations. For that, it should be possible to get access to the chart object that is generated by C3.

### PR DESCRIPTION
**Pull request by ammendonca**
Fixes #204

### CODE REVIEW

File: src/charts/c3/c3-chart.directive.js [hunk 1]
```diff
@@ -69,17 +78,23 @@
     return {
       restrict: 'A',
       scope: {
-        config: '='
+        config: '=',
+        callback: '='
```
Comment (jeff-phillips-18) [inline src/charts/c3/c3-chart.directive.js]: Can we name this as getChartCallback or generateChartCallback? In the off case where we need to add another callback.

Comment (ammendonca) [inline src/charts/c3/c3-chart.directive.js]: Agree. Will use getChartCallback.

**Comment by jeff-phillips-18** [comment]
Please add  getChartCallback to the ng-docs parameters list.

**Comment by jeff-phillips-18** [comment]
LGTM

**Comment by dtaylor113** [comment]
LGTM

**Pull request merged**